.github/workflows/update-caches.yml

```
@@ -0,0 +1,136 @@
+name: Update Actions Caches
+
+on:
+  # Manually
+  workflow_dispatch:
+  # On PR merge
+  push:
+    branches:
+      - main
+  # After nightly release
+  schedule:
+    - cron: "0 1 * * *"
+
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_PROFILE_TEST_DEBUG: 0
+  CARGO_PROFILE_DEV_DEBUG: 0
+  # If nightly is breaking CI, modify this variable to target a specific nightly version.
+  NIGHTLY_TOOLCHAIN: nightly
+
+jobs:
+  env:
+    runs-on: ubuntu-latest
+    outputs:
+      NIGHTLY_TOOLCHAIN: ${{ steps.env.outputs.NIGHTLY_TOOLCHAIN }}
+      MSRV: ${{ steps.env.outputs.MSRV }}
```

```suggestion
      MSRV: ${{ steps.msrv.outputs.MSRV }}
```

I think this needs to be `msrv`, not `env`, since this specific output is emitted from the `msrv` step.